docs/data_upload.md

```diff
@@ -53,7 +53,7 @@ https://data.scrc.uk/data_product/<namespace>:<data product name>@<version>
 ```
 and for external objects use:
 ```
-https://data.scrc.uk/external_object/<doi_or_unique_name>:<title>@<version>
+https://data.scrc.uk/external_object/<other_unique_name>:<title>@<version>
```

Comment:
I'm not sure how to resolve this because `identifier` or `other_unique_name` may be blank, and the `identifier` is a URL and will need escaping - maybe for external objects, we just need to reference its data product now?

Reply:
Yes, possibly we just drop this for external objects? Since escaping is needed this feature is somewhat less useful...

Reply:
Yes, I'd go with dropping it.

Reply:
Now uses `identifier` as download link or the above otherwise